scripts/consume_sitemaps.py

```diff
@@ -25,9 +25,7 @@
 from isb_web import sqlmodel_database
 from isb_web.sqlmodel_database import SQLModelDAO, all_thing_identifiers, thing_identifiers_from_resolved_content
 
-CONCURRENT_DOWNLOADS = 25000
-# when we hit this length, add some more to the queue
-REFETCH_LENGTH = CONCURRENT_DOWNLOADS / 2
+BATCH_SIZE = 20000
```

Comment:
Ultimately dropped the batch size down to 20k as I hit a timeout once at 25k in my local testing